deployment/terraform/modules/s3/vars.tf

```diff
@@ -2,3 +2,4 @@
 variable "bucket_name_base" {
   description = "Prefix for the Refinery Platform S3 buckets"
 }
+variable "name" {}
```

Comment:
Name parameter is only needed when creating multiple instances of a module which does not appear to be the case for any of the modules https://www.terraform.io/docs/modules/usage.html#multiple-instances-of-a-module

Reply:
got rid of it and reran apply and didn't have any problems. Pushed to branch.